src/process.cpp

```diff
@@ -585,6 +585,16 @@ std::shared_ptr<DarlingServer::Thread> DarlingServer::Process::_pickS2CThread(vo
 	return thread;
 };
 
+const std::string& DarlingServer::Process::executablePath() const {
+	std::shared_lock lock(_rwlock);
+	return _executablePath;
+};
+
+void DarlingServer::Process::setExecutablePath(const char* path) {
+	std::unique_lock lock(_rwlock);
+	_executablePath = std::string(path);
```

Comment:
Following the signature change from the header:
```suggestion
void DarlingServer::Process::setExecutablePath(std::string path) {
	std::unique_lock lock(_rwlock);
	_executablePath = path;
```